readme.md

```diff
@@ -33,28 +33,25 @@ tempWrite.sync('unicorn', 'rainbow/cake/pony.png');
 
 ## API
 
-### tempWrite(input, [filepath], callback)
+### tempWrite(input, [filepath])
+
+Returns a Promise.
 
 #### input
 
-*Required*  
```

Comment:
Don't remove these. It's linebreaks in Markdown. Install an EditorConfig plugin in your editor to abide by the .editorconfig file in this repo: http://editorconfig.org/